examples/CMakeTests.cmake

```diff
@@ -116,27 +116,6 @@ if (H5_HAVE_PARALLEL AND HDF5_TEST_PARALLEL AND NOT WIN32)
   # The number 24 corresponds to SPACE1_DIM1 and SPACE1_DIM2 defined in ph5example.c
   math(EXPR NUMPROCS "24 / ((24 + ${MPIEXEC_MAX_NUMPROCS} - 1) / ${MPIEXEC_MAX_NUMPROCS})")
 
-  set (mpi_test_ex_CLEANFILES
-  )
-  
-  # Remove any output file left over from previous test run
-  add_test (
-      NAME MPI_TEST_EXAMPLES-clear-objects
-      COMMAND    ${CMAKE_COMMAND} -E remove ${mpi_test_ex_CLEANFILES}
-  )
-  set_tests_properties (MPI_TEST_EXAMPLES-clear-objects PROPERTIES
-      FIXTURES_SETUP clear_MPI_TEST_EXAMPLES
-      WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
-  )
-  add_test (
-      NAME MPI_TEST_EXAMPLES-clean-objects
-      COMMAND    ${CMAKE_COMMAND} -E remove ${mpi_test_ex_CLEANFILES}
-  )
-  set_tests_properties (MPI_TEST_EXAMPLES-clean-objects PROPERTIES
-      FIXTURES_CLEANUP clear_MPI_TEST_EXAMPLES
-      WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
-  )
-
```

Comment:
Will these be removed permanently, or just until it is known which files should be removed.  I think the test error occurred because the list of files to delete was empty.

Reply:
I looked at the tests - they seem to cleanup properly.

Reply:
Ok, so this clean/clear code wasn't needed and needs only to be removed.